microprofile-fault-tolerance/pom.xml

```diff
@@ -200,6 +205,12 @@
             <artifactId>resteasy-jackson2-provider</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.arquillian</groupId>
+            <artifactId>arquillian-api</artifactId>
+            <version>1.0.0.Alpha5</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Remove this as this won't work when testing against deployment running in e.g. OpenShift.